selfdrive/car/gm/values.py

```diff
@@ -53,12 +54,27 @@ def __init__(self, CP):
       # lower threshold removes some braking deadzone
       max_regen_acceleration = -1. if CP.carFingerprint in EV_CAR else -0.1
 
-    self.GAS_LOOKUP_BP = [max_regen_acceleration, 0., self.ACCEL_MAX]
+    # The max amount of deceleration possible using ASCMGasRegenCmd alone.
+    # It is assumed no regen/engine braking is available at low speeds
+    self.MAX_REGEN_ACCEL_BP = [2., 10.]  # m/s
+    self.MAX_REGEN_ACCEL_V = [0., max_regen_acceleration]  # m/s^2
```

Comment:
```suggestion
    # The max amount of deceleration possible using ASCMGasRegenCmd alone.
    # Some creep acceleration is assumed at low speeds
    self.MAX_REGEN_ACCEL_BP = [2., 10.]  # m/s
    self.MAX_REGEN_ACCEL_V = [0.2, max_regen_acceleration]  # m/s^2
```

@twilsonco I think creep acceleration may be disabled if ACC is active, can you check?